Assets/_BForBoss/_Core/Scripts/Boss/DerekContextManager.cs

```diff
@@ -0,0 +1,175 @@
+using System;
+using System.Collections;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class DerekContextManager : ContextManager
+    {
+        [SerializeField, Tooltip("How long should the system wait before restarting the Labor after an incomplete Labor")]
+        private float _LaborIncompleteTimer = 5.0f;
```

Comment:
Right now, the labors (or specifically the ring system, I didn't pull the behaviour out to the general labors) already handle this themselves. I also kinda think that makes more sense, cause that way we can have time as a function of the number of rings or something like that. You'd be better of passing an event up through the ring system and listening for that rather than separately managing time here

Reply:
As long as it's
1. single source of truth.
2. makes sense within the context of the class.

Reply:
Sounds good! I like that better!